plugins/automation/app/serializers/discourse_automation/automation_serializer.rb

```diff
@@ -84,6 +85,25 @@ def trigger
       }
     end
 
+    def include_stats?
+      scope&.dig(:stats).present?
+    end
+
+    def stats
+      automation_stats = scope&.dig(:stats, object.id) || {}
+
+      {
+        last_day: automation_stats[:last_day] || empty_stats,
+        last_week: automation_stats[:last_week] || empty_stats,
+        last_month: automation_stats[:last_month] || empty_stats,
```

Comment:
I think worth considering if the null objects should be put in by `.fetch_period_summaries` for the benefit of other potential call sites.